DbTables/src/DbCache.cc

```diff
@@ -1,8 +1,18 @@
-#include <algorithm>
+
 #include "Offline/DbTables/inc/DbCache.hh"
+#include <iostream>
+#include <iomanip>
 
 void mu2e::DbCache::add(int cid, mu2e::DbTable::cptr_t const& ptr) { 
+
   _tables[cid] = ptr;
+
+  _cidFifo.push(cid);
+  _nAdded++;
+  _size += ptr->size();
+  if(_size>_hwmSize) _hwmSize = _size;
+  if(_nAdded%20==0) purge();
```

Comment:
Do we want to make "20" run-time configurable?

Reply:
I can make the 20 and the 0.9 configurable.

Reply:
Thanks.  I will watch for the commit.